### DISCUSSION TITLE
re-add option of ssh game shell

### DISCUSSION THREAD

**Discussion by NotAFile** [piqueserver/piqueserver#663] [Feature Requests]
The ssh is currently only manhole, because that's what aloha uses. But adding the option of having an administrative shell back into the ssh server would be pretty nice.

Ideally, I think it would throw you into the normal shell by default, and typing "python" or something brings you into the python manhole.

**Comment by samuelallan72** [comment]
Any reason why this couldn't be the same 'shell' accessible on the console when the server is running?

**Comment by NotAFile** [comment]
Well, that's the way it originally was. But then somewhere along the lines, someone switched to a python shell instead.

**Comment by NotAFile** [comment]
I think a good option here is to add some function that puts you into a game REPL
